src/modules/roc_rtp/validator.h

```diff
@@ -57,7 +58,7 @@ class Validator : public packet::IReader, public core::NonCopyable<> {
     packet::PacketPtr prev_packet_;
 
     const ValidatorConfig config_;
-    const size_t sample_rate_;
+    const audio::SampleSpec& sample_spec_;
```

Comment:
`const audio::SampleSpec& sample_spec_` -> `const audio::SampleSpec sample_spec_`

It's not safe to store a reference to outside value, its lifetime is unknown.